pkg/koordlet/statesinformer/states_pods.go

```diff
@@ -117,6 +120,14 @@ func (s *podsInformer) Start(stopCh <-chan struct{}) {
 			klog.Fatalf("Unable to run the pleg: ", err)
 		}
 	}()
+
+	if s.config.RecordPodMetricsInterval > 0 {
+		go wait.Until(s.recordPodMetrics, s.config.RecordPodMetricsInterval, stopCh)
```

Comment:
how about just add at the end of syncPods

Reply:
done